faq/faq.rst

```diff
@@ -14,13 +14,6 @@ How do I add nodes to an existing Citus cluster?
 
 You can add nodes to a Citus cluster by calling the master_add_node UDF with the hostname (or IP address) and port number of the new node. After adding a node to an existing cluster, it will not contain any data (shards). Citus will start assigning any newly created shards to this node. To rebalance existing shards from the older nodes to the new node, the Citus Enterprise edition provides a shard rebalancer utility. You can find more information about shard rebalancing in the :ref:`cluster_management` section.
 
-How do I change the shard count for a hash partitioned table?
```

Comment:
I'd prefer retaining this question, as it did come up somewhat frequently, rather than adding a note to the end of the other question.

Reply:
The github diff layout doesn't make it very apparent, but this question still exists in the FAQ. It's just moved below "How do I choose the shard count." Still a top-level entry in the document, though its answer is pretty brief.